picard/webservice.py

```diff
@@ -453,7 +453,9 @@ def lookup_discid(self, discid, handler, priority=True, important=True, refresh=
         return self._get_by_id('discid', discid, handler, inc, queryargs={"cdstubs": "no"},
                                priority=priority, important=important, refresh=refresh)
 
-    def _find(self, entitytype, handler, kwargs):
+    def _find(self, entitytype, handler, kwargs,
```

Comment:
You could move the arguments that are only passed to `self.get` to kwargs here as well. Also, are there plugins that make use of this function? AFAICT all users set `priority` and `important` to `True`, so the defaults should reflect that.